base-api/Startup.cs

```diff
@@ -112,22 +112,21 @@ public void ConfigureServices(IServiceCollection services)
 
         private static void ConfigureDbContext(IServiceCollection services)
         {
-            //TODO: Rename Context and connection string to match Database
-            var connectionString = Environment.GetEnvironmentVariable("UH_URL");
+            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
 
             var builder = new DbContextOptionsBuilder().UseSqlServer(connectionString);
```

Comment:
Wouldn't that need to use '.UseNpgsql', rather than 'UseSqlServer'?

Reply:
Yes - thank you for spotting this.